src/documentation/pages/components-wip/dropdown/examples/_icon_trigger.vue

```diff
@@ -0,0 +1,76 @@
+<template lang="pug">
+<ComponentExample title="Icon" id="icon_trigger" :tabs="exampleTabs">
+  template(#example-description)
+    p.-text
+      | There is an available property, <code>icon</code>, which allows adding an icon as the dropdown trigger.
+  template(#example)
+    .chi-dropdown
+      button#example__dropdown-icon.chi-button.chi-dropdown__trigger.chi-dropdown__icon(aria-label="More options" data-tooltip="More options")
```

Comment:
In vue docs we can use the web component. For portal theme, in the website documentation it's being used  `variant=flat`. In this docs you can access the selected theme with `selectedTheme = useSelectedTheme();` and use the right variant: `:variant="selectedTheme === 'portal' ? 'flat' : ''".

Code snippets should also reflect this change.